superset-frontend/src/views/CRUD/data/common.ts

```diff
@@ -39,5 +39,11 @@ export const commonMenuData = {
       url: '/savedqueryview/list/',
       usesRouter: true,
     },
+    {
+      name: 'Query History',
+      label: t('Query History'),
```

Comment:
Is "Query History" the same as "Query Search"? I thought query search is for searching all saved queries across the whole company that is not necessarily authored by you?